core/kazoo_endpoint/src/kz_endpoint.erl

```diff
@@ -1131,6 +1131,7 @@ create_sip_endpoint(Endpoint, Properties, #clid{}=Clid, Call) ->
                       ,{<<"Failover">>, maybe_build_failover(Endpoint, Clid, Call)}
                       ,{<<"Metaflows">>, kz_json:get_json_value(<<"metaflows">>, Endpoint)}
                       ,{<<"Endpoint-Actions">>, endpoint_actions(Endpoint, Call)}
+                      ,{<<"Exclude-From-Queues">>, kz_json:get_value(<<"exclude_from_queues">>, Endpoint)}
```

Comment:
This should not be a new root level property but should be in the existing object:

"qubicle": {
    "enabled": true,
   "presence_id": "7900@test.com",
    "exclude": true
}